server/util/networking/networking_test.go

```diff
@@ -189,6 +186,22 @@ func TestContainerNetworking(t *testing.T) {
 	}
 }
 
+func TestAllowTrafficToHostDefaultIP(t *testing.T) {
+	testnetworking.Setup(t)
+	flags.Set(t, "executor.task_allowed_private_ips", []string{"default"})
+	ctx := context.Background()
+	err := networking.EnableMasquerading(ctx)
+	require.NoError(t, err)
+	defaultIP, err := networking.DefaultIP(ctx)
+	require.NoError(t, err)
+
+	// Create container network
+	c1 := createContainerNetwork(ctx, t)
+
+	// Should be able to reach host's default IP when flag is enabled
+	netnsExec(t, c1.NamespacePath(), fmt.Sprintf("ping -c 1 -W 1 %s", defaultIP))
+}
```

Comment:
Credit to @dan-stowell for this test - from https://github.com/buildbuddy-io/buildbuddy/commit/a48207375f61195bb49e66cb3fea093140c3b6f4